src/backend/distributed/executor/multi_utility.c

```diff
@@ -188,6 +188,14 @@ multi_ProcessUtility(Node *parsetree,
 		return;
 	}
 
+	if (!CitusHasBeenLoaded())
+	{
+		standard_ProcessUtility(parsetree, queryString, context,
```

Comment:
I thought about adding it but couldn't come up with anything that added to the code. Isn't that what the code already says?